cfme/middleware/servers.py

```diff
@@ -0,0 +1,30 @@
+from cfme.fixtures import pytest_selenium as sel
+from cfme.web_ui.menu import nav
+from cfme.middleware.provider import HawkularProvider
+from . import list_tbl
+
+nav.add_branch(
+    'middleware_servers',
+    {
+        'middleware_servers_detail':
+            lambda ctx: list_tbl.click_cell('Provider', ctx['provider'].name)
+    }
+)
+
+
+class Servers(HawkularProvider):
```

Comment:
I believe these classes should not inherit from HawkularProvider as they are not HawkularProvider. They are objects that are related to the provider, but not derived from ...